addon/components/lt-body.js

```diff
@@ -239,6 +290,55 @@ export default Component.extend({
     this.set('useVirtualScrollbar', fixedHeader || fixedFooter);
   },
 
+  didUpdateAttrs({ newAttrs, oldAttrs }) {
+    this._super(...arguments);
+
+    let newScrollTo = get(newAttrs, 'scrollTo.value');
+    let oldScrollTo = get(oldAttrs, 'scrollTo.value');
+    let newScrollToRow = get(newAttrs, 'scrollToRow.value');
+    let oldScrollToRow = get(oldAttrs, 'scrollToRow.value');
```

Comment:
Yeah. I spoke with locks and confirmed that we should not be using the arguments in `didUpdateAttrs`. Sorry for making you revert 😅